### PR TITLE
Token management spring stack tutorial

### PR DESCRIPTION
**Pull request by dogeared**

### CODE REVIEW

File: extensions/spring/boot/docs/source/tutorial.rst [hunk 1]
```diff
@@ -519,6 +519,146 @@ Add the custom data to one of the users, but not the other.
...
+
+In order to hit the `/me` endpoint, we'll first, we'll get an `access_token` and a `refresh_token` by hitting the
+`/oauth/token` endpoint:
+
+.. code-block:: bash
+
+    curl -v -X POST \
+      -H "Origin: http://localhost:8080" \
+      -H "Content-Type: application/x-www-form-urlencoded" \
+      -d "grant_type=password&username=<valid email address>&password=<valid password>" \
```
Comment (mrioan) [inline extensions/spring/boot/docs/source/tutorial.rst]: <valid_email_address> and <valid_password>

File: extensions/spring/boot/docs/source/tutorial.rst [hunk 2]
```diff
@@ -519,6 +519,146 @@ Add the custom data to one of the users, but not the other.
...
+    {
+      "access_token":"eyJraWQiOiJSOTJTQkhKQzFVNERBSU1HUTNNSE9HVk1YIiwiYWxnIjoiSFMyNTYifQ.eyJqdGkiOiI1eDlxbWlES2U0RmlFMU02alhLSDBMIiwiaWF0IjoxNDU0NDQ0MTU1LCJpc3MiOiJodHRwczovL2FwaS5zdG9ybXBhdGguY29tL3YxL2FwcGxpY2F0aW9ucy82dkZUNEFSZldDbXVIVlY4Vmt0alRvIiwic3ViIjoiaHR0cHM6Ly9hcGkuc3Rvcm1wYXRoLmNvbS92MS9hY2NvdW50cy80V1NjTWJBbm8zVjk1aWlTc3dralBYIiwiZXhwIjoxNDU0NDQ3NzU1LCJydGkiOiI2M1laa01xMTlzYUhxTHZqSDFtbzRLIn0.J2NR7MV3OoolYImfUNiu8SCDvaQdresHTnPHgL7mO1Q",
+      "refresh_token":"eyJraWQiOiJSOTJTQkhKQzFVNERBSU1HUTNNSE9HVk1YIiwiYWxnIjoiSFMyNTYifQ.eyJqdGkiOiI2M1laa01xMTlzYUhxTHZqSDFtbzRLIiwiaWF0IjoxNDU0NDM4MTQ3LCJpc3MiOiJodHRwczovL2FwaS5zdG9ybXBhdGguY29tL3YxL2FwcGxpY2F0aW9ucy82dkZUNEFSZldDbXVIVlY4Vmt0alRvIiwic3ViIjoiaHR0cHM6Ly9hcGkuc3Rvcm1wYXRoLmNvbS92MS9hY2NvdW50cy80V1NjTWJBbm8zVjk1aWlTc3dralBYIiwiZXhwIjoxNDU5NjIyMTQ3fQ.yK5twgj3-v51z4pszKXWTX9VtCbs1KxQU4vH1eXvgGo",
+      "token_type":"Bearer",
+      "expires_in":3600
+    }
+
+While the `refresh_token` is the same, we get a new `access_token`.
+
+By default, when you logout, the `access_token` will be revoked. Let's see this in action:
```
Comment (mrioan) [inline extensions/spring/boot/docs/source/tutorial.rst]: ...both the `access_token` and its associated `refresh_token` will be revoked

File: tutorials/spring-boot/05-token-management/build.gradle [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.2.RELEASE")
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'spring-boot'
+
+group = 'com.stormpath'
+version = '0.1.0'
```
Comment (mrioan) [inline tutorials/spring-boot/05-token-management/build.gradle]: not familiar with gradle, `version = 0.1.0` is intended? What is this for?

File: tutorials/spring-boot/05-token-management/src/main/java/com/stormpath/tutorial/Application.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * @since 1.0.RC8.3
+ */
+@SpringBootApplication
+public class Application  {
+    public static void main(String[] args) {
```
Comment (mrioan) [inline tutorials/spring-boot/05-token-management/src/main/java/com/stormpath/tutorial/Application.java]: line break between 25 and 26 please.

File: tutorials/spring-boot/05-token-management/src/main/java/com/stormpath/tutorial/config/SpringSecurityWebAppConfig.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+import static com.stormpath.spring.config.StormpathWebSecurityConfigurer.stormpath;
+
+/**
+ * @since 1.0.RC8.3
+ */
+@Configuration
+public class SpringSecurityWebAppConfig extends WebSecurityConfigurerAdapter {
+    @Override
```
Comment (mrioan) [inline tutorials/spring-boot/05-token-management/src/main/java/com/stormpath/tutorial/config/SpringSecurityWebAppConfig.java]: line break between 28 and 29 please.

File: tutorials/spring-boot/05-token-management/src/main/java/com/stormpath/tutorial/controller/MeController.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @since 1.0.RC8.3
+ */
+@RestController
+public class MeController {
+    @RequestMapping(value="/me", produces = MediaType.APPLICATION_JSON_VALUE)
```
Comment (mrioan) [inline tutorials/spring-boot/05-token-management/src/main/java/com/stormpath/tutorial/controller/MeController.java]: line break between 31 and 32 please.

File: tutorials/spring-boot/05-token-management/src/main/java/com/stormpath/tutorial/model/AccountInfo.java [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stormpath.tutorial.model;
+
+/**
+ * @since 1.0.RC8.3
+ */
+public class AccountInfo {
+    private String href;
```
Comment (mrioan) [inline tutorials/spring-boot/05-token-management/src/main/java/com/stormpath/tutorial/model/AccountInfo.java]: line break between 21 and 22 please.

**Comment by mrioan** [comment]
@dogeared back to you. If you agree with all the proposed changes and your next commit passes in Travis (last one failed) then you can consider this PR approved. Thanks!

File: tutorials/spring-boot/05-token-management/build.gradle (hunk 3 above)
Comment (dogeared) [inline tutorials/spring-boot/05-token-management/build.gradle]: yeah. It's similar to project version in pom.xml. It doesn't auto-update on deploy though.

**Comment by dogeared** [comment]
back to you, @mrioan! Thanks.

**Pull request merged**